app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import os
+
+
+def move_file(command_directory_file: str) -> None:
+    data = command_directory_file.split()
+    if len(data) == 3 and data[0] == "mv" and data[1] != data[2]:
```

Comment:
For more descriptive code, perform unpacking in the future instead of indexing for value retrieval